En_Decoder.cpp

```diff
@@ -13,62 +13,97 @@ void helper()
 	qDebug() << "Available commands:";
     qDebug() << ">1 ENCODE";
     qDebug() << ">2 DECODE";
-    qDebug() << ">5 HELP";
     qDebug() << ">0 EXIT";
 }
 
-int main()
+int main(int argc, char *argv[])
 {
-    QTextStream qtin(stdin);
-    QString line = qtin.readLine();
-    QString dictionaryPath;
-    qDebug() << "Pls give me the path: ";
-    qtin >> dictionaryPath;
+    QCoreApplication app(argc, argv);
+    QCoreApplication::setApplicationName("En_Decoder");
+    QCoreApplication::setApplicationVersion("0.8.0");
+
+    //qDebug() << app.arguments();
+
+    QCommandLineParser parser;
```

Comment:
positional argument dict.json path
print help if encode or decode option not set